ts/parts-map/HeatmapSeries.ts

```diff
@@ -461,11 +508,45 @@ seriesType<Highcharts.HeatmapSeriesOptions>(
 
             // Get the extremes from the y data
             Series.prototype.getExtremes.call(this);
+        },
+
+        /**
+         * Mapping object between `inactive.except` and built-in methods
+         */
+        inactiveFilters: {
+            all: function (): boolean {
+                return true;
+            },
+            column: function (
+                activePoint: Highcharts.HeatmapPoint,
+                otherPoint: Highcharts.HeatmapPoint
+            ): boolean {
+                return activePoint.x === otherPoint.x;
+            },
+            'column-row': function (
+                activePoint: Highcharts.HeatmapPoint,
+                otherPoint: Highcharts.HeatmapPoint
+            ): boolean {
+                return activePoint.x === otherPoint.x ||
+                    activePoint.y === otherPoint.y;
+            },
+            row: function (
+                activePoint: Highcharts.HeatmapPoint,
+                otherPoint: Highcharts.HeatmapPoint
+            ): boolean {
+                return activePoint.y === otherPoint.y;
+            },
+            self: function (
+                activePoint: Highcharts.HeatmapPoint,
+                otherPoint: Highcharts.HeatmapPoint
+            ): boolean {
+                return false;
```

Comment:
In this case you could remove the function parameters as they do not serve any purpose. Or is this a temporary thing?

Reply:
Good point! I'm not convinced if we even need this filter, added because it seems to be quite short one. Will update args.